apiserver/src/api/error.rs

```diff
@@ -3,16 +3,9 @@ use models::node::BottlerocketShadowError;
 use actix_web::error::ResponseError;
 use snafu::Snafu;
 
-/// The crate-wide result type.
-pub type Result<T> = std::result::Result<T, Error>;
-
-/// The crate-wide error type.
 #[derive(Debug, Snafu)]
 #[snafu(visibility(pub))]
-pub enum Error {
-    #[snafu(display("Unable to create client: '{}'", source))]
-    ClientCreate { source: kube::Error },
-
+pub enum ApiError {
```

Comment:
Do you have preferences and any thoughts that make you keep this `error.rs`? I think might be better to delete this file and move the ApiError module to `mod.rs`? I'm not 100% confident that my way is better than yours. Love to hear your thoughts! thanks